### DISCUSSION TITLE
i forgor they retire the b29 stuperfortress

### DISCUSSION THREAD

**Discussion by rwg100** [3kh0/ext-remover#383] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
i cant nuke 3kho no more :(

**Comment by Aka-but-nice** [comment]
You have to  use The Boeing-B-52 that thing is powerful💀

**Reply by rwg100** [reply]
yeah but dropping nukes with it is illegal :(

**Comment by 3kh0** [comment]
Why are you trying to nuke me

**Reply by Aka-but-nice** [reply]
Just because,
